release_templates/vintage/aws/v1alpha3/cluster/awscluster.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: infrastructure.giantswarm.io/v1alpha3
```

Comment:
I think for AWS this makes more sense because we have to wire so much config together currently.